### ISSUE TITLE
Improvement: Publish aws-appmesh-proxy-route-manager image in all App Mesh regions

### ISSUE THREAD

**Issue by lavignes** [aws/aws-app-mesh-roadmap#279]
*If you want to see App Mesh implement this idea, please upvote with a :+1:.*

**Tell us about your request**
The `aws-appmesh-proxy-route-manager` image only exists in the `us-west-2` region:

```
840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-proxy-route-manager
```

App Mesh should publish this image in all supported regions as it does with the Envoy image.

**Which integration(s) is this request for?**
EC2 and K8s mainly.

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**

Some customers cannot or do not wish to make cross-region requests to fetch images.

**Are you currently working around this issue?**

Not sure how others are working around it. Some users may be replicating images to other regions or creating exceptions to pull from ECR in us-west-2.

**Comment by anand-rajagopal-hub** [comment]
For private clusters, not having the image in all the regions (us-east-1 in my case) makes it really difficult. Have to manually get the image, upload it to ECR within the account, and pass in the value to helm chart. Would be easier to have the image in all regions

**Comment by tanvp112** [comment]
How do we know if the version in the helm chart is the latest? Currently the envoy image (1.17.2) is behind the latest (1.19).

**Comment by suniltheta** [comment]
@tanvp112  Updating the version that you see on the helm chart is independent effort from the App Mesh Envoy image release process. Now if you see the latest available AppMesh k8s Controller version `v1.4.1` the default sidecar image is listed as `v1.16.1.1-prod` (link: [GitHub](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/v1.4.1/config/helm/appmesh-controller/values.yaml#L18)). But the latest available App Mesh Envoy image is `v1.20.0.0` (see: public ECR [page](https://gallery.ecr.aws/appmesh/aws-appmesh-envoy)). The next release for AppMesh k8s Controller will be update to use the latest available side car image at that point in time.
  
However this doesn't restrict the image that we can use. We can set `sidecar.image.tag` config variables on the controller to pick the envoy version that we want. 
  ```
  --set sidecar.image.tag=v1.20.0.0-prod
  ```


So to answer your question: >> *How do we know if the version in the helm chart is the latest?*  
  Always refer App Mesh ECR public repo https://gallery.ecr.aws/appmesh/aws-appmesh-envoy for info on latest envoy image. If this is not what you find in the helm chart you can always set `sidecar.image.tag` to the latest available.

**Comment by suniltheta** [comment]
The `aws-appmesh-proxy-route-manager` image have been released in all regions. To use this image in your k8s deployment you can update your App Mesh k8s controller config to set an `init.image.repository` and `init.image.tag`.

```bash
--set init.image.repository= 840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-proxy-route-manager \
--set init.image.tag=v4-prod
```
The above command 
#### To pick region specific images:

* For `me-south-1`:
  ```
  772975370895.dkr.ecr.me-south-1.amazonaws.com/aws-appmesh-proxy-route-manager:v4-prod
  ```
* For `ap-east-1`:
  ```
  856666278305.dkr.ecr.ap-east-1.amazonaws.com/aws-appmesh-proxy-route-manager:v4-prod
  ```
* For `eu-south-1`:
  ```
  422531588944.dkr.ecr.eu-south-1.amazonaws.com/aws-appmesh-proxy-route-manager:v4-prod
  ```
* For `af-south-1`:
  ```
  924023996002.dkr.ecr.af-south-1.amazonaws.com/aws-appmesh-proxy-route-manager:v4-prod
  ```
* For `cn-north-1`:
  ```
  919366029133.dkr.ecr.cn-north-1.amazonaws.com.cn/aws-appmesh-proxy-route-manager:v4-prod
  ```
* For `cn-northwest-1`:
  ```
  919830735681.dkr.ecr.cn-northwest-1.amazonaws.com.cn/aws-appmesh-proxy-route-manager:v4-prod
  ```

* For all other [regions where App Mesh is available](https://docs.aws.amazon.com/general/latest/gr/appmesh.html):
  ```
  840364872350.dkr.ecr.<region>.amazonaws.com/aws-appmesh-proxy-route-manager:v4-prod
  ```

  e.g.
  ```
  840364872350.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-proxy-route-manager:v4-prod
  ```
